cddl/examples/ext-veraison-json-2.diag

```diff
@@ -0,0 +1,25 @@
+{
+  "eat_profile": "tag:github.com,2023:veraison/ear",
+  "iat": 1666529184,
+  "ear.verifier-id": {
+    "developer": "https://veraison-project.org",
+    "build": "vts 0.0.1"
+  },
+  "ear.raw-evidence": "NzQ3MjY5NzM2NTYzNzQK",
+  "submods": {
+    "PARSEC_TPM": {
```

Comment:
Should be `TPM_PARSEC` -- to be consistent with `TPM_ENACTTRUST`?

Reply:
good question :-)

I "think" in this case it should be `PARSEC_TPM`, because Parsec acts as the RoT abstraction and could (in the future) drive a PSA backend too, and others as well.

So `PARSEC_PSA`, `PARSEC_DICE`, etc.

Reply:
Hm, in that case, shouildn't it just be `PARSEC`? How is the backend relevant here?

Reply:
The underlying evidence formats are different: what Parsec does is it provides a single  API to request key (and platform) attestations, and uses a common encapsulation for the produced evidence.  But the encapsulated evidence is still in its native format and needs, so different RoT backends will be associated to different Veraison schemes.